README.md

```diff
@@ -12,9 +12,15 @@ Sequelize is a promise-based Node.js/io.js ORM for Postgres, MySQL, MariaDB, SQL
 
 `npm install sequelize`
 
-Will install the latest version of sequelize (currently 2.0).
+From 3.0.0 and up Sequelize will follow SEMVER.
 
-[Upgrading to 2.0](https://github.com/sequelize/sequelize/wiki/Upgrading-to-2.0)
+## Backwards compatibility changes
```

Comment:
Hm I think this will prevent a lot of people opening tickets for BC. There are so many people who code without knowing anything about semver and things like that. I mean, it won't hurt leaving it, right?